src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+document.addEventListener('contextmenu', (ev) => {
+  ev.preventDefault();
+});
+
+const first = new Promise((resolve, reject) => {
+  document.addEventListener('click', (ev) => {
+    if (ev.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+const second = new Promise((resolve) => {
+  document.addEventListener('mousedown', (ev) => {
+    if (ev.button === 0 || ev.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
```

Comment:
The 'second' promise resolves on both left and right mouse button presses due to the condition `ev.button === 0 || ev.button === 2`. This might cause it to resolve unexpectedly if both buttons are pressed. Ensure this behavior is intended.